packages/block-library/src/navigation-link/style.scss

```
@@ -2,7 +2,7 @@
 	display: flex;
 	align-items: center;
 	position: relative;
-	margin: 0;
+	margin: 0 8px 0 0;
```

Could we do this in ems instead? 
```suggestion
	margin: 0 0.5em 0 0;
```

Since all other blocks use `em` values instead of `px` (with the exception of social links), I think it would make sense to keep it consistent